src/components/Alerts/__tests__/UpsellBanner.test.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import toJson from 'enzyme-to-json';
+import UpsellBanner from '../UpsellBanner';
+
+describe('UpsellBanner component', () => {
+
+    it('should render', () => {
+        const wrapper = shallow(<UpsellBanner/>);
+        expect(toJson(wrapper)).toMatchSnapshot();
+    });
+
+    /* eslint-disable no-console */
+    it('should click close with default prop', () => {
+        console.log = jest.fn();
```

Comment:
You can test in here, that the component does not fail without any `onClose` function instead of checking console.log